index.js

```diff
@@ -12,6 +12,55 @@ server.get('/users', (req, res) => {
         .catch(() => {
             res.status(500).json({ error: "The users information could not be retrieved."})
         })
-})
+});
+
+server.get('/users/:id', (req, res) => {
+    const id = req.params.id;
+    // const {id} = req.params;
+    db.findById(id)
+    .then(response => {
+        if (response.length < 1) {
+            res.status(404).json({ message: "The user with the specified ID does not exist."})
+        } else {
+            res.json(response)
+        }
+    })
+    .catch(() => {
+        res.status(500).json({ error: "The user information could not be retrieved."})
+    })
+});
+
+server.delete('/users/:id', (req, res) => {
+    const id = req.params.id;
+    // const {id} = req.params
+    db.remove(id)
+    .then(erase => {
+        if (erase > 0) {
+            res.status(200).json(erase)
+        } else {
+            res.status(404).json({ message: "The user with the specified ID does not exist."})
+        }
+    })
+    .catch(() => {
+        res.status(500).json({ error: "The user could not be removed" })
+    })
+});
+
+server.post('/users', (req, res) => {
+    const user = req.body;
+    if (user.name === null || user.bio === null) {
```

Comment:
Here you can actually shorten your code a little bit by using not operators (!):    if (!name || !bio)